testsuite/tests/cxx-api/all_includes.ml

```diff
@@ -1,13 +1,41 @@
 (* TEST
- modules = "stubs.c";
- readonly_files = "all-includes.h";
- not-msvc;
- flags = "-ccopt -x -ccopt c++ -ccopt -std=c++11";
+ has-cxx;
```

Comment:
Just a note, as I puzzled myself - this test skips on AppVeyor because the testsuite runs bytecode-only (this test could be explicitly done with the dynlink ones, but I don't think it's worth it?)